platform/event/event_payload.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2024 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import "state_signature_payload.proto";
+
+option java_package = "com.swirlds.proto.legacy.event";
+// <<<pbj.java_package = "com.swirlds.proto.event">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * The payload of an event, which is either generated by the application or the platform
+ */
+message EventPayload {
+  /**
+   * the payload
+   */
```

Comment:
comments on the `oneof` are generally not retained in specification/documentation output.  Prefer to specify the fields instead.

Reply:
ok, removed this